### PR TITLE
[159]Create & Style the Customer Signup Page using CSS-Frontend

### PR DESCRIPTION
**Pull request by mutmainaho**
screencapture-127-0-0-1-8080-html-forms-signup-html-2021-01-07-16_47_52
screencapture-127-0-0-1-8080-html-forms-signup-html-2021-01-03-13_41_41
screencapture-127-0-0-1-8080-html-forms-signup-html-2021-01-03-13_42_00

### CODE REVIEW

**Comment by Ireti203** [comment]
The font size on mobile looks too big. Kindly reduce it.

File: submissions/MutmainahOdunewu/html-forms/signup.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <title>signup page</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <h2><i>sandalsbymo</i></h2>
+        <ul class="look">
+            <li><a href="../HTML/index.html">Product Page</a></li>
+            <li><a href="../html-forms/contact.html">Contact Page</a></li>
+            <li><a href="./signup.html"></a>Sign Up</li>
+
```
Comment (Ireti203) [inline submissions/MutmainahOdunewu/html-forms/signup.html:16]: Remove whitespace

File: submissions/MutmainahOdunewu/html-forms/signup.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          <input type="text" name="phonenumber" id="phonenumber"/><br><br>
+          <label for="emailaddress">Email Address</label><br>
+          <input type="email" name="emailaddress" id="emailaddress"/><br><br>
+          <label for="password">Password</label><br>
+          <input type="password" name="password" id="password"/><br><br>
+          <label for="confirm">Confirm Password</label><br>
+          <input type="password" name="confirm" id="confirm"/><br><br>
+          <input type="checkbox" name="agree" id="agree" value="agree"/>
+          <label for="agree" id="lab">I agree to the Terms and Conditions</label><br><br>
+          <button type="submit" id="bttn">SIGN UP</button>
```
Comment (Ireti203) [inline submissions/MutmainahOdunewu/html-forms/signup.html:43]: This should be under the form tag, format properly. Also, remove the breaks and style with css instead.

File: submissions/MutmainahOdunewu/html-forms/signup.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+           <li><a href="#"><img src="../HTML/images/twitterlogo.png"></a></li>
+           <li><a href="#"><img src="../HTML/images/gmaillogo.png"></a></li>
+           <li><a href="#"><img src="../HTML/images/youtubelogo.png"></a></li>
+        </ul>
+  </div>
+</div>  
+<p id="copyright">&copy; Copyright 2020 sandalsbymo.com <span> All rights reserved</span></p>
+</footer>
+    
+
```
Comment (Ireti203) [inline submissions/MutmainahOdunewu/html-forms/signup.html:76]: Remove whitespace

File: submissions/MutmainahOdunewu/html-forms/signup.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/signup.css">
+    <title>signup page</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <h2><i>sandalsbymo</i></h2>
+        <ul class="look">
```
Comment (Ireti203) [inline submissions/MutmainahOdunewu/html-forms/signup.html:12]: Kindly use a more descriptive naming for the class.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by adaugochi** [comment]
According to the mockup, the label i.e `first name, last name` etc are not aligned to the centre. Kindly change that to align left

**Pull request closed without merging**